packages/teams-js/src/public/app.ts

```diff
@@ -178,12 +178,12 @@ export namespace app {
    */
   export interface AppHostInfo {
     /**
-     * The name of the host client. Possible values are: Office, Orange, Outlook, Teams
+     * Identifies which host is running your app
```

Comment:
Do we use second person pronouns elsewhere in the documentation? Also, I think we've used "application" rather than "app".

Reply:
Discussed offline. This is recommended language. Approving.